### DISCUSSION TITLE
WWDC24 위너 Organization 초대 안내

### DISCUSSION THREAD

**Discussion by x-0o0** [wwdc-kr/submissions#24] [초대요청]
🎉 WWDC24 Swift Student Challenge 에 위너로 선발되신 것을 축하드립니다

WWDC Scholars Korea 에 초대되지 않으신 분들은 현재 discussion에 이메일을 댓글로 남겨주세요.

위너 선발 여부 확인 후 바로 초대 됩니다.

**Comment by dasoya** [comment]


**Reply by x-0o0** [reply]
> ekth020216@gmail.com
> 
> 연락 부탁드립니다. 

메일 확인 부탁드립니다~

**Comment by Haewooo** [comment]
eunuk2002@gmail.com 
연락 부탁드립니다.

**Reply by x-0o0** [reply]
인스타 DM 확인 부탁드립니다

**Comment by shine-jung** [comment]
joydonald5314@gmail.com
연락 부탁드립니다~!

**Reply by x-0o0** [reply]
확인 및 초대 완료
